core/src/Cuda/Kokkos_Cuda_KernelLaunch.hpp

```diff
@@ -410,6 +410,8 @@ struct CudaParallelLaunch< DriverType
 
       KOKKOS_ENSURE_CUDA_LOCK_ARRAYS_ON_DEVICE();
 
+      printf("invoke functor on device: %ld \n", sizeof(driver));
```

Comment:
It's not polite for Kokkos to print something on every single process, every time a kernel launches.